nntrainer/graph/network_graph.cpp

```diff
@@ -281,6 +294,9 @@ int NetworkGraph::addLossLayer(const LossType loss_type) {
   layer->input_layers.clear();
   layer->input_layers.push_back(input_str);
 
+  std::shared_ptr<LossLayer> temp = std::dynamic_pointer_cast<LossLayer>(layer);
```

Comment:
use createLoss directly than using createLayer at line 284.

Reply:
Actually I intended not to include *_layer.h file here. But once I includes them to support like ActivationLayer::type, I can create loss layer directly. done.